components/function-runtimes/python39/kubeless/kubeless.py

```diff
@@ -116,6 +116,13 @@ def publishCloudEvent(self, data):
             headers = {"Content-Type": "application/cloudevents+json"}
             )
     
+    def resolvedatatype(event_data):
+        match type(event_data)
+            case 'dict':
+                return 'application/json'
+            case 'str':
+                return 'text/plain'
```

Comment:
is it possible to receive type other than dict or str? what in a situation like this? This question is for two other switch sentences too

Reply:
There is no such possibility thus we don't need a default case.